packages/components/pie-modal/src/index.ts

```diff
@@ -1,14 +1,23 @@
-import { LitElement, html } from 'lit'; // eslint-disable-line import/no-extraneous-dependencies
+import { LitElement, html, unsafeCSS } from 'lit'; // eslint-disable-line import/no-extraneous-dependencies
 import { RtlMixin } from '@justeattakeaway/pie-webc-core';
 
+import styles from './modal.scss?inline';
+
 export class PieModal extends RtlMixin(LitElement) {
     // eslint-disable-next-line class-methods-use-this
     render () {
         return html`
-            <div>This is the Pie Modal
-                <slot></slot>
-            </div>`;
+            <dialog class="o-modal" open>
+                <h3 class="o-modal-header">Modal header</h3>
```

Comment:
I'd probably go with `heading` - `header` could be used if we ever have a header block for the modal in future. 

```suggestion
                <h3 class="o-modal-heading">Modal header</h3>
```